frontend/app/templates/components/mapbox/carto-vector-source.hbs

```diff
@@ -7,7 +7,7 @@
     {{! Add the source }}
     <MapboxGlSource
       @map={{map.instance}}
-      @sourceId={{this.elementId}}
+      @sourceId={{if this.sourceId this.sourceId this.elementId}}
```

Comment:
Any particular reason to manually setting and tracking the `sourceId`? 
Just wondering, I'm not opposed to it. Does it help with debugging? 
I remember @allthesignals mentioning that it used the elementId so that it doesn't have to be manually set/tracked.

Reply:
Yeah, if you check out https://github.com/NYCPlanning/ceqr-app/pull/405/files#diff-b5d9eb53bd5a6e7394282f46bf25bfd5R58 you'll see that part of the hack to make sure we display the bbl geom as top-most layer requires checking that the bbls layer and the other layers exist. Thought it was better to explicitly name them, rather than check for sources with ids like 'ember###' and assume that those were the right ones. what if we add more sources! I do wonder what you mean by "tracking" tho.. what tracking is necessary of an explicitly named map source? all of the sources in the maps have real names (from the basemap) except these ember ones.

Reply:
👍 Ahhh nice! Looks awesome.

Oh I just meant like we have to think of names of each of the sources, and make sure we don't name two sources in a map the same thing.